.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
```

Comment:
This file maybe got committed prior to me adding .vagrant to the .gitignore; I'd remove it.